docs/content/getting_started/prerequisites.md

```diff
@@ -10,6 +10,6 @@ Make sure you have the following available before proceeding :
 - Access to Databricks environment with access to Delta tables/databases
 - Ability to create Databricks job and run them on new cluster
 - Proper access permissions to create database, tables and write data to the desired OMS location
-- Procured the Delta OMS solution package from your Databricks team (includes DeltaOMS jar , 
-sample notebooks and documentations)
+- Access to create [libraries](https://github.com/pureconfig/pureconfig) on your Databricks environment. Required to attach the relevant DeltaOMS (delta-oms) and PureConfig maven `com.github.pureconfig:pureconfig_2.12:0.14.0` libraries
```

Comment:
@himanishk can you try making next release without `pureconfig` requirement? databricks runtime already has some config management libraries. The less third-party jars required - the faster adoption will be

Reply:
Thanks @nfx . Will explore for next release to keep things simple.